hazelcast/src/test/java/com/hazelcast/internal/util/LatencyDistributionTest.java

```diff
@@ -52,44 +52,50 @@ public void accuracyTest() {
         }
 
         double accuracyFactor = (actual * 1.0d / expected);
-        assertTrue("accuracyFactor = " + accuracyFactor, accuracyFactor < 1.01);
+        assertTrue("accuracyFactor = " + accuracyFactor, accuracyFactor < 1.39);
```

Comment:
So actually, with the changes our accuracy decreases slightly. But 1.xx is still an acceptable accuracy I guess, given it's in us.

Reply:
I don't understand why we need to give up on accuracy.

Reply:
I think accuracy was calculated wrongly previously, now it should be ok now.